.github/workflows/dbt_slim_ci.yml

```
@@ -0,0 +1,48 @@
+# .github/workflows/app.yaml
+name: dbt slim ci (in beta)
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - 'main'
+
+concurrency:
+  group: ${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  dbt-test:
+    runs-on: [ self-hosted, linux, spellbook ]
+    timeout-minutes: 60
+
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v2
+
+      - name: Add git_sha to schema
+        env:
+          GIT_SHA: ${{ github.ref }}
+        run: sed -ie 's!^\(\s*schema:\)[^.]*!\1 git_$GIT_SHA!' profiles.yml
```

```suggestion
        run: sed -ie 's!^\(\s*schema:\)[^.]*!\1 git_$GIT_SHA!' profiles.yml
```
        run: "/runner/change_schema.sh git_$GIT_SHA"